content/docs/studio/user-guide/projects-and-experiments/run-experiments.md

```diff
@@ -16,24 +16,108 @@ your model training using Iterative Studio.
 
 Iterative Studio uses your regular CI/CD setup (e.g. GitHub Actions) to run the
 experiments. This means that to enable experimentation from Iterative Studio,
-you need to integrate your Git repository with a CI/CD setup (e.g. GitHub
-Actions).
+you need to do the following:
 
-Then, on each Git commit, the CI/CD setup will get invoked. If this setup
-includes a model training process, it will be triggered, which means that your
-ML experiment will run automatically.
+1. First, integrate your Git repository with a CI/CD setup that includes model
+   training process. You can do this in one of two ways:
 
-For more details on how to set up
-[CI/CD pipelines for your ML project](/doc/use-cases/ci-cd-for-machine-learning),
-refer to [CML](https://cml.dev). You can use any cloud or Kubernetes for the
-model training process. CML also generates a report after the CI/CD setup
-executes.
+   - by
+     [using the wizard provided by Iterative Studio](#use-the-iterative-studio-wizard-to-set-up-your-ci-action).
+   - by creating the CI set up on your own. For this, create workflow files
+     (such as GitHub Actions) that get triggered on push or pull request. For
+     more details on how to set up
+     [CI/CD pipelines for your ML project](/doc/use-cases/ci-cd-for-machine-learning),
+     refer to [CML](https://cml.dev).
```

Comment:
This (#1) is pretty much a repeat of [Prepare Your Repositories to Run New Experiments](https://dvc-org-cml-in-studio-bgeulj3d.herokuapp.com/doc/studio/user-guide/prepare-your-repositories#prepare-your-repositories-to-run-new-experiments). Why not just link instead?

Reply:
Done.